bazel/test_constants.bzl

```diff
@@ -31,21 +31,29 @@ TAG_MANUAL = ["manual"]
 
 # Used for a "sudo test" that needs to be executed by a user with sudo
 # privileges. This is a restriction mainly known for some Python tests.
+# To run sudo tests execute: $MAGMA_ROOT/bazel/scripts/run_sudo_tests.sh
 # Note: for now a sudo test is also tagged as "manual".
 TAG_SUDO_TEST = ["sudo_test"] + TAG_MANUAL
 
 # Used for integration tests. These tests need to be executed manually
 # by a user with sudo privileges. These tags represent test categories,
 # which are used to determine the appropriate environment for them.
-TAG_PRECOMMIT_TEST = ["precommit_test"] + TAG_MANUAL
-TAG_EXTENDED_TEST = ["extended_test"] + TAG_MANUAL
-TAG_EXTENDED_TEST_SETUP = ["extended_setup"] + TAG_MANUAL
-TAG_EXTENDED_TEST_TEARDOWN = ["extended_teardown"] + TAG_MANUAL
-TAG_NON_SANITY_TEST = ["nonsanity_test"] + TAG_MANUAL
-TAG_NON_SANITY_TEST_SETUP = ["nonsanity_setup"] + TAG_MANUAL
-TAG_NON_SANITY_TEST_TEARDOWN = ["nonsanity_teardown"] + TAG_MANUAL
+# To run the LTE integration tests execute: $MAGMA_ROOT/bazel/scripts/run_integ_tests.sh
+TAG_INTEGRATION_TEST = ["integration_test"]
+TAG_PRECOMMIT_TEST = ["precommit_test"] + TAG_MANUAL + TAG_INTEGRATION_TEST
+TAG_EXTENDED_TEST = ["extended_test"] + TAG_MANUAL + TAG_INTEGRATION_TEST
+TAG_EXTENDED_TEST_SETUP = ["extended_setup"] + TAG_MANUAL + TAG_INTEGRATION_TEST
+TAG_EXTENDED_TEST_TEARDOWN = ["extended_teardown"] + TAG_MANUAL + TAG_INTEGRATION_TEST
+TAG_NON_SANITY_TEST = ["nonsanity_test"] + TAG_MANUAL + TAG_INTEGRATION_TEST
+TAG_NON_SANITY_TEST_SETUP = ["nonsanity_setup"] + TAG_MANUAL + TAG_INTEGRATION_TEST
+TAG_NON_SANITY_TEST_TEARDOWN = ["nonsanity_teardown"] + TAG_MANUAL + TAG_INTEGRATION_TEST
 TAG_TRAFFIC_SERVER_TEST = ["traffic_server_test"]
 
+# Used for load tests. These tests need to be executed manually
+# by a user with sudo privileges.
```

Comment:
Could we change this to: `These tests need to be executed by the '$MAGMA_ROOT/bazel/scripts/run_load_tests.sh' script to preserve the ordering of the test cases.`